src/js/commands/italic.js

```diff
@@ -14,10 +14,21 @@ export default class ItalicCommand extends TextFormatCommand {
     this.markup = builder.createMarkup('em');
   }
   exec() {
-    this.editor.applyMarkupToSelection(this.markup);
+    let markerRange = this.editor.cursor.offsets;
+    if (!markerRange.leftRenderNode || !markerRange.rightRenderNode) {
+      return;
+    }
```

Comment:
This feels like inside baseball. The italic command should never be `exec`-able if there isn't a selection — if the right/left render nodes are not present then something else is wrong, or the command is being `exec`'d when there's no selection to apply markup to.